bliss/catalog.py

```diff
@@ -149,12 +149,7 @@ def galaxy_bools(self) -> Tensor:
         return is_galaxy * self.is_on_mask.unsqueeze(-1)
 
     @property
-    def on_fluxes(self):
-        """Gets fluxes of "on" sources based on whether the source is a star or galaxy.
-
-        Returns:
-            Tensor: a tensor of fluxes of size (b x nth x ntw x max_sources x 1)
-        """
+    def on_nmgy(self):
```

Comment:
So internally are we storing everything in nanomaggies now, even for DC2?

Reply:
(Will that change how the encoder operates on DC2?)

Reply:
Yes, at the beginning of this project, we stored fluxes in nanomaggies.